modules/tracker/models/dao/ScalarDao.php

```diff
@@ -25,32 +25,13 @@
  * @method void setScalarId(int $scalarId)
  * @method int getSubmissionId()
  * @method void setSubmissionId(int $submissionId)
+ * @method Tracker_SubmissionDao getSubmission()
  * @method int getTrendId()
```

Comment:
should there also be 

`* @method void setSubmission(Tracker_SubmissionDao $submissionDao)`

?